ui/config/environment.js

```diff
@@ -15,7 +15,7 @@ module.exports = function(environment) {
     EmberENV: {
       FEATURES: {
         // Here you can enable experimental features on an ember canary build
-        // e.g. 'with-controller': true
+        // e.g. EMBER_NATIVE_DECORATOR_SUPPORT: true
         'ember-routing-router-service': true,
```

Comment:
This diff, which I imagine was programmatic, reminds me that we are still explicitly enabling the router service. Surely that's not necessary anymore, yeah?

Reply:
True!!! I’ve removed it now, thanks.